evm/db/state.py

```diff
@@ -205,15 +205,17 @@ def delete_storage(self, address):
     #
     # Balance
     #
-    def get_balance(self, address):
-        validate_canonical_address(address, title="Storage Address")
+    def get_balance(self, address, validate=True):
```

Comment:
It is not clear to me why we need a flag here to disable validation.  First impression is that this is not a good idea as it allows invalid data to get into the state db.

Reply:
We do not need it.  Initially I thought I would have to step around the validation in order for the set_balance to work with an arbitrary non-existent address. What was really happening, is the address I had been using was in fact not valid.  I'll take this out.

Does the arbitrary address that I hardcoded have the risk of colliding with a real address?  Are addresses generated (with eth_account) but never used in an on-chain transaction safely considered unique?